base/commands/viridian/custom_class_it_test.go

```diff
@@ -0,0 +1,93 @@
+package viridian_test
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/it"
+)
+
+func customClassTest(t *testing.T) {
+	viridianTester(t, func(ctx context.Context, tcx it.TestContext) {
+		// setup
+		f := "foo.zip"
+		fd := "testdata/" + f
+		cID := ensureClusterRunning(ctx, tcx)
+		// test upload custom class
+		tcx.WithReset(func() {
```

Comment:
I think  `upload-custom-class` and others should have separate tests instead of collecting them all in `customClassTest`.

Reply:
why do you think so? I find it easy to test especially for CRUD operations.